x-pack/test/ui_capabilities/security_and_spaces/tests/catalogue.ts

```diff
@@ -148,15 +148,15 @@ export default function catalogueTests({ getService }: FtrProviderContext) {
             expect(uiCapabilities.value!.catalogue).to.eql(expected);
             break;
           }
+          case 'global_read at nothing_space':
```

Comment:
__question:__ do you need this change? I believe these `xxx at nothing_space` should stay where they were, under the corresponding `the nothing_space has no Kibana features enabled,....` comment.